packages/fether-react/src/stores/parityStore.js

```diff
@@ -18,10 +20,11 @@ const electron = isElectron() ? window.require('electron') : null;
 const LS_KEY = `${LS_PREFIX}::secureToken`;
 
 export class ParityStore {
-  @observable
-  downloadProgress = 0;
   @observable
   isApiConnected = false;
+  isApiConnected$ = timer(0, 1000).pipe(
+    map(_ => this.api && this.api.isConnected)
+  );
```

Comment:
Slightly more Rx would be to add `distinctUntilChanged()` here, and remove `if (isApiConnected === this.isApiConnected)`

Reply:
Actually this.isApiConnected isn't used anywhere, I'll just ditch it